### PR TITLE
Loss

### PR DESCRIPTION
**Pull request by rashchedrin**

### CODE REVIEW

File: dann_config.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+weight_domain_loss = 1
+weight_prediction_loss = 1 # todo: find actual values
+unk_value = -100 # torch default
+target_domain_idx = 1
```
Comment (dudevil) [inline dann_config.py]: перевод строки в конце файла

File: dann_config.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+weight_domain_loss = 1
```
Comment (dudevil) [inline dann_config.py]: давайте константы заглавными буквами делать

File: dann_loss.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import torch
+import dann_config
+import numpy as np
```
Comment (dudevil) [inline dann_loss.py]: лучше поменять местами: https://www.python.org/dev/peps/pep-0008/#imports

File: dann_loss.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+def loss_DANN_(
```
Comment (dudevil) [inline dann_loss.py]: если хочется сказать, что это внутренний метод то лучше loss_DANN_ -> _loss_DANN

File: dann_loss.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    
+    crossentropy = torch.nn.CrossEntropyLoss(ignore_index=unk_value)
+    prediction_loss = crossentropy(class_predictions_logits, instances_labels)
+    binary_crossentropy = torch.nn.BCEWithLogitsLoss()
+    domain_loss = binary_crossentropy(logprobs_target, is_target)
+    loss = weight_domain_loss * domain_loss \
+         + weight_prediction_loss * prediction_loss
+    return loss
+
+def test_loss_DANN_():
```
Comment (dudevil) [inline dann_loss.py:39]: тесты это круто! Но давай их в отдельной директории склывать? типа tests/  в корне

File: dann_loss.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                [1.0,2,3],
+                [4,5,6],
+                [7,8,9],
+                [10,11,12]
+            ]))
+    lpt = torch.Tensor(np.array([-5.1,5,-6, 8]))
+    ils = np.array([1,2,-100,2], dtype='int')
+    ist = np.array([0,1,0,1], dtype='int')
+    assert(abs(0.7448 - loss_DANN_(cpl, lpt, ils, ist)) < 1e-4)
+
```
Comment (dudevil) [inline dann_loss.py]: между верхнеуровневыми функциями нужно 2 пустые строки
